booking/models.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+import uuid
+
+
+class Event:
+
+    def __init__(self, title, total_ticket_count, ticket_price,
+                 sale_end, sale_start,
+                 max_tickets_in_order=50,
+                 order_expiration_period=timedelta(minutes=30)):
+        self._id = uuid.uuid4().hex
+        self.title = title
+        self.total_ticket_count = total_ticket_count
+        self.ticket_price = ticket_price
+        self.sale_start = sale_start
+        self.sale_end = sale_end
+        self.max_tickets_in_order = max_tickets_in_order
+        self.order_expiration_period = order_expiration_period
+
+    @property
```

Comment:
Probably, there is no need to use property, because for now it can be defined as attribute. Then when we will use database probably given primary key will be enough.

Reply:
But using property prevents from setting it from the outside. There's not `setter` here.